LICENSE.note

```diff
@@ -0,0 +1 @@
+This software is in the public domain because it contains materials that originally came from the United States Geological Survey, an agency of the United States Department of Interior. For more information, see the official USGS copyright policy at http://www.usgs.gov/visual-id/credit_usgs.html#copyright
```

Comment:
general comment: we also have been putting this preliminary message in `.onAttach()`, such as here: https://github.com/USGS-R/gsplot/blob/67aa258e3c9efe4c8886559415e74932bc666bcc/R/gsplot.R#L2